assets/elm/main.elm

```diff
@@ -54,18 +54,22 @@ type Msg
     | Table String
     | NameTagChange NameTag
     | UpdateState Json.Encode.Value
+    | StartGame Topic
+    | UpdateGame Json.Encode.Value
+    | ProgressGame Topic
+    | ChooseCategory Topic
```

Comment:
No need to have all these state change Msg constructors take a Topic as well. They can just read the `model.tableTopic` in the `update` handler

Reply:
Good point